protobufs/funding/funding.proto

```diff
@@ -30,6 +30,14 @@ service FundingService {
       description: "Get a funding agreement of a document"
     };
   }
+  rpc GetList(GetListRequest) returns (FundingListResponse) {
+    option (google.api.http) = {
+      get: "/documents/{identifier}/extension/funding"
```

Comment:
I understand the extension on the codebase. Dont think we need to put it under URL IMO

Reply:
I don't have a strong opinion about it.  We could also change it.